src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1045,7 +1045,7 @@ describe('ReactDOMComponent', () => {
         ReactDOM.render(<div is="custom-div" />, container);
         expect(document.createElement).toHaveBeenCalledWith(
           'div',
-          'custom-div',
+          {is: 'custom-div'},
```

Comment:
`document.createElement` should not have a second argument of a string, but rather an `options` object